packages/sources/coinpaprika-test/src/config/index.ts

```diff
@@ -0,0 +1,10 @@
+export const DEFAULT_API_ENDPOINT = 'https://api.coinpaprika.com'
+export const PRO_API_ENDPOINT = 'https://api-pro.coinpaprika.com'
+
+export const customSettings = {
+  API_KEY: {
+    description: 'An API key Coinpaprika',
+    type: 'string',
+    required: false,
+  },
```

Comment:
We should mark this config as sensitive so it gets redacted in logs
```suggestion
  API_KEY: {
    description: 'An API key Coinpaprika',
    type: 'string',
    required: false,
    sensitive: true
  },
```